src/containers/Sidebar/SidebarBottomItem.js

```diff
@@ -31,7 +28,6 @@ const SidebarBottomItem = ({ item, sidebarLogo }) => {
 		// or if item should always be uncollapsed
 		else if (item.children && !uncollapseAll) setOpen(prev => !prev);
 		// Close small sidebar on item click
```

Comment:
@Aringoaway You left the comment here, which is now unrelated. Remove it (also in the other file)